src/lib/apollo/paginateDepleteProjectsQuery.ts

```diff
@@ -15,25 +17,9 @@ export async function paginateDepleteProjectsQueryCall({
 }: {
   variables: QueryProjectsArgs
 }) {
-  const first = variables.first ?? 1000
-  const skip = variables.skip
-
-  const { data } = await serverClient.query<ProjectsQuery, QueryProjectsArgs>({
-    query: ProjectsDocument,
-    variables: { ...variables, first, skip },
+  return paginateDepleteQuery<ProjectsQuery, QueryProjectsArgs>({
```

Comment:
this is just a wrapper for the frequently-used projects-`paginateDepleteQuery` that makes reuse cleaner